src/io/flutter/run/common/CommonTestConfigUtils.java

```diff
@@ -89,7 +87,7 @@ private void clearCachedInfo(String path) {
     final String path = file.getVirtualFile().getPath();
     final boolean outlineOutdated;
     synchronized (this) {
-      outlineOutdated = pathToPreviouslyUsedOutline.containsKey(path) && outline != pathToPreviouslyUsedOutline.get(path);
+      outlineOutdated = cache.containsKey(path) && outline != cache.get(path).outline;
```

Comment:
nit: the cleaner pattern for this in java is
final entry = cache.get(path);

Reply:
Done.